README.md

```diff
@@ -17,6 +17,8 @@
 
 **check-ebs-snapshots.rb**
 
+**check-ec2-filter.rb**
```

Comment:
This needs to get added to the files section as well.  Right now they're basically a dupe.  We need to go through at some point and add the descriptions of what the plugins do to the repo.